File: 백준/Bronze/2445. 별 찍기 － 8/별 찍기 － 8.py
n = int(input())

for i in range(1, n + 1):
    print("*" * i +  " " * (n * 2 - i * 2) + "*" * i)
for i in range(n - 1, 0, -1):
    print("*" * i +  " " * (n * 2 - i * 2) + "*" * i)

File: 백준/Bronze/2445. 별 찍기 － 8/별 찍기 － 8.js
const path = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
const input = require('fs').readFileSync(path).toString().trim();

const number = Number(input);

for (let i = 1; i <= number; i++) {
  console.log('*'.repeat(i) + ' '.repeat((number - i) * 2) + '*'.repeat(i));
}

for (let i = number - 1; i >= 1; i--) {
  console.log('*'.repeat(i) + ' '.repeat((number - i) * 2) + '*'.repeat(i));
}
